ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx

```diff
@@ -521,12 +534,30 @@ function validateUtmUps(key: string, value: any) {
   return error
 }
 function validateRadiusLineBuffer(key: string, value: any) {
+  const parsed = Number(value.value)
+  const buffer = Number.isNaN(parsed) ? 0 : parsed
+  const bufferMeters = DistanceUtils.getDistanceInMeters(buffer, value.units)
+  if (key === 'lineWidth' && bufferMeters < 75) {
```

Comment:
might be worth adding a comment for posterity on why 75 was chosen...  "75 meters determined to be a reasonable min for linestring searches based on data source capabilities"  (or similar)

Reply:
Done.